### PR TITLE
add apicurio

### PR DESCRIPTION
**Pull request by paoloantinori**
relates to https://github.com/operate-first/apps/pull/506

### CODE REVIEW

**Review by tumido — COMMENTED** [review]
/lgtm

**Comment by paoloantinori** [comment]
There seems to be some error with PROW preventing me to check the logs. I don't know if that was the error in first instance:

https://prow.operate-first.cloud/

**Comment by HumairAK** [comment]
@paoloantinori we're currently upgrading the cluster, so the CI is in a bit of a funny state, we'll post here when the upgrade is complete: https://github.com/operate-first/SRE/issues/269 
And also in the slack announcement channel

File: envs/moc/zero/apicurio/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - registry.yml
```
Comment (HumairAK) [inline envs/moc/zero/apicurio/kustomization.yaml:4]: CI was likely failing because there's no newline in this yaml.

**Comment by HumairAK** [comment]
You can also run the linting pre-commit tests locally: https://github.com/operate-first/apps/blob/master/contributing.md#tests

**Comment by paoloantinori** [comment]
> commit

thank you. done now. the `pre-commit` invocation fixed that but also printed some errors I'm not sure they are relevant:
```
Traceback (most recent call last):
  File "/home/pantinor/.cache/pre-commit/repoc8j_xnr8/py_env-python3/bin/check-added-large-files", line 8, in <module>
    sys.exit(main())
  File "/home/pantinor/.cache/pre-commit/repoc8j_xnr8/py_env-python3/lib/python3.9/site-packages/pre_commit_hooks/check_added_large_files.py", line 62, in main
    return find_large_added_files(
  File "/home/pantinor/.cache/pre-commit/repoc8j_xnr8/py_env-python3/lib/python3.9/site-packages/pre_commit_hooks/check_added_large_files.py", line 33, in find_large_added_files
    filenames_filtered = set(filenames) - lfs_files()
  File "/home/pantinor/.cache/pre-commit/repoc8j_xnr8/py_env-python3/lib/python3.9/site-packages/pre_commit_hooks/check_added_large_files.py", line 21, in lfs_files
    return set(json.loads(lfs_ret)['files'])
  File "/usr/lib64/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Review by tumido — COMMENTED** [review]
/ok-to-test
/lgtm

**Review by tumido — APPROVED** [review]
/approve

**Comment by tumido** [comment]
Pre-commit issue encountered above seems to be a local issue. Probably caused by https://github.com/pre-commit/pre-commit-hooks/issues/252 This doesn't affect the PR itself.

**Pull request merged**